metricbeat/module/vsphere/virtualmachine/data.go

```diff
@@ -78,6 +79,36 @@ func (m *MetricSet) mapEvent(data VMData) mapstr.M {
 	if len(data.triggeredAlarms) > 0 {
 		event.Put("triggered_alarms", data.triggeredAlarms)
 	}
+	if val, ok := data.PerformanceData["cpu.usage.average"]; ok {
+		event.Put("cpu.usage.percent", val)
+	}
+	if val, ok := data.PerformanceData["disk.usage.average"]; ok {
+		intVal, assertOk := val.(int64)
+		if assertOk {
+			event.Put("disk.average.bytes", intVal*kilobytesToBytesMultiplier)
+		}
+	}
+	if val, ok := data.PerformanceData["disk.read.average"]; ok {
+		intVal, assertOk := val.(int64)
+		if assertOk {
+			event.Put("disk.read.average.bytes", intVal*kilobytesToBytesMultiplier)
+		}
+	}
+	if val, ok := data.PerformanceData["disk.write.average"]; ok {
+		intVal, assertOk := val.(int64)
+		if assertOk {
+			event.Put("disk.write.average.bytes", intVal*kilobytesToBytesMultiplier)
+		}
+	}
+	if val, ok := data.PerformanceData["disk.numberRead.summation"]; ok {
+		event.Put("disk.numberRead", val)
```

Comment:
```suggestion
		event.Put("disk.numberRead.count", val)
```
WDYT, is this better, aligning with the namings in host?

Reply:
I agree, made the change